Windows builds broken since rust 1.64

@schtauffen in Discord had problem installing lunatic-runtime today and I'm assuming it is picking up 0.10.0
```
    error[E0277]: the trait bound file_type::FileType: std::sealed::Sealed is not satisfied
    <snip>
        |
    134 | impl std::os::windows::fs::FileTypeExt for FileType {
        |      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ the trait std::sealed::Sealed is not implemented for file_type::FileType
        |
    <snip>
    error: could not compile cap-primitives due to previous error
    warning: build failed, waiting for other jobs to finish...
    error: failed to compile lunatic-runtime v0.10.0, intermediate artifacts can be found at <snip>

```

I went through the :carrot:  :hole: :rabbit2: on this and I suspect (slightly) wasi is broken in Windows via latest Rust stable 1.64.0

There was some trait changes that landed in 1.64.0 and traced it here: https://github.com/rust-lang/rust/issues/102260

This is a TODO Issue

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -80,3 +80,12 @@ members = [
     "crates/lunatic-registry-api",
     "crates/lunatic-distributed-api",
 ]
+
+##########################################################
+# This is required until wasmtime is bumped to ^1
+# https://github.com/bytecodealliance/wasmtime/issues/4936
+# Tracking: https://github.com/lunatic-solutions/lunatic/issues/136
+[patch.crates-io]
+cap-std = { git = "https://github.com/bytecodealliance/cap-std.git", branch = "0.26-with-a-0.25.2-version-number" }
+cap-primitives = { git = "https://github.com/bytecodealliance/cap-std.git", branch = "0.26-with-a-0.25.2-version-number" }
+cap-fs-ext = { git = "https://github.com/bytecodealliance/cap-std.git", branch = "0.26-with-a-0.25.2-version-number" }

